src/test/java/reposense/authorship/AnnotatorAnalyzerTest.java

```diff
@@ -0,0 +1,84 @@
+package reposense.authorship;
+
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.Date;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import reposense.authorship.model.FileInfo;
+import reposense.authorship.model.FileResult;
+import reposense.authorship.model.LineInfo;
+import reposense.git.GitClone;
+import reposense.model.Author;
+import reposense.model.RepoConfiguration;
+import reposense.model.RepoLocation;
+import reposense.util.TestUtil;
+
+public class AnnotatorAnalyzerTest {
```

Comment:
Why don't you sub-class from `GitTestTemplate` which already has the testrepo-alpha cloned?

You can have a look at `FileInfoExtractorTest` class to see how test cases are done

Reply:
Done